clientd3d/d3drender_objects.c

```diff
@@ -1204,7 +1204,7 @@ void D3DRenderOverlaysDraw(
 					}
 
 					*gameObjectDataParams.numObjects++;
-				TEMP_END2: // ewww we have a goto.. todo: lets remove
```

Comment:
You can certainly reword this if you want, but having a TODO to remove a goto seems eminently reasonable.